frontend/src/metabase/dashboard/hoc/types.ts

```diff
@@ -1,46 +1,14 @@
 import type { Location } from "history";
-import type { LocationAction } from "react-router-redux";
 
-import type { EmbeddingDisplayOptions } from "metabase/public/lib/types";
+import type { DashboardDisplayOptionControls } from "metabase/dashboard/hoc/controls/use-dashboard-display-options";
 import type { DashboardId } from "metabase-types/api";
 
-import type { FetchDashboardResult } from "../types";
-
 // passed via ...this.props
 export type DashboardControlsProps = {
   location: Location;
-  fetchDashboard: (opts: {
-    dashId: DashboardId;
-    queryParams?: Record<string, unknown>;
-    options?: {
-      clearCache?: boolean;
-      preserveParameters?: boolean;
-    };
-  }) => Promise<FetchDashboardResult>;
+  queryParams: Record<string, string | string[] | null | undefined>;
   dashboardId: DashboardId;
-  fetchDashboardCardData: (opts?: {
-    reload?: boolean;
-    clearCache?: boolean;
-  }) => Promise<void>;
-};
-
-// Passed via ...this.state
-export type DashboardControlsState = {
-  isFullscreen: boolean;
-  refreshPeriod: number | null;
-  theme: EmbeddingDisplayOptions["theme"];
-  hideParameters: string;
 };
 
-export type DashboardControlsPassedProps = {
-  // Passed via explicit props
-  replace: LocationAction;
-  isNightMode: boolean;
-  hasNightModeToggle: boolean;
-  setRefreshElapsedHook: (hook: (elapsed: number) => void) => void;
-  loadDashboardParams: () => void;
-  onNightModeChange: (isNightMode: boolean) => void;
-  onFullscreenChange: (isFullscreen: boolean) => void;
-  onRefreshPeriodChange: (refreshPeriod: number | null) => void;
-} & DashboardControlsProps &
-  DashboardControlsState;
+export type DashboardControlsPassedProps = DashboardDisplayOptionControls &
+  DashboardControlsProps & { loadDashboardParams: () => void };
```

Comment:
`loadDashboardParams` is still being used in the main `Dashboard` app, and I don't want to touch it for now